apps/btshell/src/cmd.c

```diff
@@ -2203,9 +2203,11 @@ cmd_conn_datalen(int argc, char **argv)
 
 #if MYNEWT_VAL(SHELL_CMD_HELP)
 static const struct shell_param conn_datalen_params[] = {
-    {"conn", "conn_datalen handle, usage: =<UINT16>"},
-    {"octets", "usage: =<UINT16>"},
-    {"time", "usage: =<UINT16>"},
+    {"conn", "Connection handle, usage: =<UINT16>"},
+    {"octets", "Max payload size to include in LL Data PDU, "
+               "range=<0x001B-0x00FB>, usage: =<UINT16>"},
```

Comment:
decimal numbers would be so much nicer here

Reply:
Agreed, fixed.